File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Book/BillingContact.vb
Imports Newtonsoft.Json

Namespace ExpediaRapid.SerializableClasses.Book

    Public Class BillingContact

        <JsonProperty("given_name")>
        Public Property GivenName As String

        <JsonProperty("family_name")>
        Public Property FamilyName As String

        <JsonProperty("email")>
        Public Property Email As String

        <JsonProperty("phone")>
        Public Property Phone As Phone

        <JsonProperty("address")>
        Public Property Address As Address
    End Class

End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/Book/BillingContact.cs
namespace iVectorOne.Suppliers.ExpediaRapid.SerializableClasses.Book
{
    using Newtonsoft.Json;


    public class BillingContact
    {

        [JsonProperty("given_name")]
        public string GivenName { get; set; }

        [JsonProperty("family_name")]
        public string FamilyName { get; set; }

        [JsonProperty("address")]
        public Address Address { get; set; }
    }

}